medicines/api/src/azure_search.rs

```diff
@@ -90,12 +90,12 @@ pub fn create_context() -> AzureContext {
 }
 
 impl AzureSearchClient {
-    pub async fn azure_search(
-        &self,
-        search_term: String,
-    ) -> Result<AzureSearchResults, reqwest::Error> {
-        azure_search(search_term, &self.client, &self.config).await
-    }
+    // pub async fn azure_search(
```

Comment:
instead of doing this can we use `#[allow(dead_code)]`?
https://doc.rust-lang.org/rust-by-example/attribute/unused.html